cmake/Modules/Finddune-uggrid.cmake

```diff
@@ -0,0 +1,34 @@
+include (OpmPackage)
+
+find_opm_package (
+  # module name
+  "dune-uggrid"
+
+  # dependencies
+  # TODO: we should probe for all the HAVE_* values listed below;
```

Comment:
did you mean to list something?

Reply:
Not sure. Seems like nothing is needed despite the VERSION defines if used.